### ISSUE TITLE
Details for new binary releases

### ISSUE THREAD

**Issue by Caoimhinmg** [PmagPy/PmagPy#151]
Hey @moonshoes87 so now we are post workshop and there was so much release branch stuff going on I'm curious how we're going to do this next binary release? I am assuming we are still going to use release in the same way we did initially? For now I'm going to create a release branch the same way we did, as a cc of master and start testing then push if everything goes all right and start compiling. Thoughts or reasons to hold the release?

**Comment by moonshoes87** [comment]
Yes, that sounds good.  I will move forward with the OSX binaries.

**Comment by moonshoes87** [comment]
We do need to fix #152 before releasing for pip.  I'm going to work on one of Lisa's bugs for the moment.  

If you are planning to make the release branch, I won't make one.  Let me know if you are not making the branch.

**Comment by Caoimhinmg** [comment]
Yeah I'll handle the release branch I'm working on some testing bugs I found for now, should be up by 5 at the absolute latest. I'll also fix #152 in a bit.

**Comment by moonshoes87** [comment]
Ok, no hurry.  I’ll work on other things, and plan to do the OSX releases tomorrow.

**Comment by Caoimhinmg** [comment]
Guessing we're going for 2.0.3 release version? Going to upload the Windows 32 binaries now.

**Comment by ltauxe** [comment]
bearing in mind that there are still lots of bugs in pmag gui….
lisa

**Comment by moonshoes87** [comment]
There are some issues with the native 3.0 conversions and everything fitting together.  @Caoimhinmg , I’m going to keep working on this a bit on the release branch.  I'll probably ask for some help from you as well, but this may all have to wait for tomorrow.

**Comment by Caoimhinmg** [comment]
Okay now we're making changes to the release branch, and I didn't get the message about the 3.0 problems until after I had made a Windows release. (sorry about that) I suppose we should up the next release version number to 2.0.4 because well there is already a Windows 2.0.3 from yesterday.

**Comment by moonshoes87** [comment]
I made an OSX release before I realized some bugs, too.  Let’s just keep in communication and we can try to get the next one out soon.

**Comment by Caoimhinmg** [comment]
Okay I know this is not amazingly important considering the small amount of our users based on Linux systems, but pyinstaller is completely shattered in Linux in both the latest release and the dev branch. It won't make a .spec file (thing used to create specifications to compile off of) or compile off of the .spec files I have saved. So there might be a bit of a wait on the new Linux binary I'm creating a bug report for pyinstaller's dev team now.

**Comment by moonshoes87** [comment]
I'm going to go ahead with a release for OSX and pip today.  I don't know if everything is fixed, but based on what I've done over the last few days (see #155) I am very confident that this release will be less buggy than the one before it.  

@Caoimhinmg, since your release process takes longer, it would be great if you could double check me and play around with Pmag GUI a little extra before you make your releases, and I want to get the pip/OSX releases out even if we end up re-releasing again shortly.  

@ltauxe, I still *think* I have probably fixed some of your troubles with Pmag GUI/Thellier GUI, but it is hard to know for sure.  I can re-release pretty quickly if needed.  

I'm going to be traveling tomorrow, hence the push to release my parts today, but after that I should be reachable!

**Comment by Caoimhinmg** [comment]
Will do, I'll run all tests and then do some manual testing on a number of my test data sets. Also I have fixed the Pyinstaller problem first error I got was just a bug the developers missed and caused about 4 days ago, the second error had something to do with my multi-arch build of skype??? I don't get it, but purging the program and installing the new beta release of skype for linux 64 only fixed it and I learned a new pdb trick. So I should be able to have a linux and windows release up next week. I will comment here when It's done and then if everything else is out I'll close this thread.

**Comment by moonshoes87** [comment]
Great.  My releases are up.  @Caoimhinmg, when yours are done let me know and I'll merge the release branch back into master.  I would rather handle this merge because of some of the recent changes I've made!

**Comment by njarboe** [comment]
Thanks all for the new stand-alones. Hopefully I can get Gary Scott at the
BGC using it for his data in the next few weeks.

**Comment by moonshoes87** [comment]
Release branch merged into master.  It was a little messy to do the merge, but I think everything is there.  Had to rebase and force push again, hopefully this doesn't cause you any problems @Caoimhinmg.  I am hoping to close out this release branch without any more new commits, because the last merge was messy, but I'm leaving it until I can verify that a few more things work correctly.

**Comment by moonshoes87** [comment]
@Caoimhinmg, I made another fix on master so merged master back into release.  sigh.  sorry for all the git messiness.

**Comment by Caoimhinmg** [comment]
@moonshoes87 It's fine, I'm travelling today and going to be busy for a few days so sorry about the delay on the releases on my end. I'll try to finish them as soon as possible. In the mean time there is no need to merge commits from master to release, unless everything is really broken and we're making a new release, as the idea of us coordinating version numbers and using a release branch is so we release the same code even if it's a bit buggy.

**Comment by Caoimhinmg** [comment]
All right everyone the new Windows and Linux binaries have been released and are functioning over here at least.
